src/LecueNote/page/LeceuNotePage/index.tsx

```diff
@@ -1,14 +1,26 @@
+import { useState } from 'react';
+
 import Header from '../../../components/common/Header';
 import CreateNote from '../../components/CreateNote';
 import Footer from '../../components/Footer';
 import * as S from './LecueNotePage.style';
 
 function LecueNotePage() {
+  const MAX_LENGTH = 1000;
+  const [contents, setContents] = useState('');
+
+  const handleChangeContents = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setContents(e.target.value);
+    if (e.target.value.length > MAX_LENGTH) {
+      setContents((e.target.value = e.target.value.substring(0, 1000)));
```

Comment:
p2: 상수화 한 김에 
```suggestion
      setContents((e.target.value = e.target.value.substring(0, MAX_LENGTH)));
```
요로케 해 보는 건 어떨까요~~?

Reply:
헉 ; ; ; 놓친 부분 이슈 ; ; ; ;감사용 ㅎㅎㅎ